internal/cli/plugin/plugin.go

```diff
@@ -15,15 +15,37 @@
 package plugin
 
 import (
+	"errors"
 	"fmt"
+	"strings"
 
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/plugin"
 	"github.com/spf13/cobra"
 )
 
 type Opts struct {
-	plugins []*plugin.Plugin
+	plugins          []*plugin.Plugin
+	existingCommands []*cobra.Command
+}
+
+func (opts *Opts) findPluginWithArg(arg string) (*plugin.Plugin, error) {
```

Comment:
Could you add a small comment on top of this function describing what it does? "With arg" is a very generic name.

Having some examples of what arg parameter could look like would improve readability by a lot.

Reply:
done